test/parallel/test-tls-cli-min-version-1.2.js

```diff
@@ -0,0 +1,15 @@
+// Flags: --tls-min-v1.2
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto) common.skip('missing crypto');
+
+// Check that node `--tls-min-v1.2` is supported.
+
+const assert = require('assert');
+const tls = require('tls');
+
+assert.strictEqual(tls.DEFAULT_MAX_VERSION, 'TLSv1.3');
+assert.strictEqual(tls.DEFAULT_MIN_VERSION, 'TLSv1.2');
+
+// Check the min-max version protocol versions against these CLI settings.
+require('./test-tls-min-max-version.js');
```

Comment:
As with the other PR, totally not blocking, but using `require()` to load another test file does give me pause...

Reply:
Note that it is the same pattern as the other 4 or 5 tests of TLS CLI options.

Reply:
I agree with @Trott that this does not seem ideal. I would move the `test-tls-min-max-version.js` file into the fixtures folder and require that instead.

Reply:
I can see the argument for it not being a fixture. It's test code, not a fixture loaded by a test. And so we want it linted and so on....

Maybe there's some way to abstract it into `common`? That's probably more trouble than it's worth.

Anyway, it gives me pause when I see it, but I'm OK with it, especially since it's the way it's already being done....

Reply:
(I am going to spend half a second wondering if it should be a fixture every single time I come across it though. 😀)